sdk/src/Handlers/EntityFramework/AWSXRayInterceptorExtensions.cs

```diff
@@ -0,0 +1,40 @@
+﻿//-----------------------------------------------------------------------------
+// <copyright file="TraceableSqlCommand.net45.cs" company="Amazon.com">
+//      Copyright 2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+//      Licensed under the Apache License, Version 2.0 (the "License").
+//      You may not use this file except in compliance with the License.
+//      A copy of the License is located at
+//
+//      http://aws.amazon.com/apache2.0
+//
+//      or in the "license" file accompanying this file. This file is distributed
+//      on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+//      express or implied. See the License for the specific language governing
+//      permissions and limitations under the License.
+// </copyright>
+//-----------------------------------------------------------------------------
+
+using Microsoft.EntityFrameworkCore;
+
+namespace Amazon.XRay.Recorder.Handlers.EntityFramework
```

Comment:
For extensions like this we generally put them in the same namespace as the type we are putting the extension method on. In this case the namespace should be `Microsoft.EntityFrameworkCore`. That way the methods light up as soon as the add the NuGet package. This was done in X-Ray's extension methods for [IApplicationBuilder](https://github.com/aws/aws-xray-sdk-dotnet/blob/master/sdk/src/Handlers/AspNetCore/AWSXRayMiddlewareExtensions.cs).

Reply:
Will change it later